### DISCUSSION TITLE
Athena integration

### DISCUSSION THREAD

**Discussion by MatejNedic** [awspring/spring-cloud-aws#34] [Ideas]
Currently there is no integration for Athena service and it would be really cool if we would integrate it. 
https://aws.amazon.com/athena/
https://docs.aws.amazon.com/code-samples/latest/catalog/code-catalog-java-example_code-athena.html
https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/athena/AmazonAthena.html#tagResource-com.amazonaws.services.athena.model.TagResourceRequest-

**Comment by eddumelendez** [comment]
Similar to the proposal for [Neptune intergation](https://github.com/awspring/spring-cloud-aws/discussions/36#discussioncomment-368590). Let's define the scope of this
